### PR TITLE
Frontend review component

### PR DESCRIPTION
**Pull request by vvndn**
image

Added review components and menu item page

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -17,13 +19,13 @@ class App extends Component {
   }
 
   render() {
-    const { menuItems } = this.state;
+    const { menuItems } = this.state; // eslint-disable-line
     return (
       <Switch>
         <Route exact path="/" component={Loginpage} />
         <Route exact path="/recommend" component={Recommendpage} />
+        <Route exact path="/menuitem" component={MenuItempage} />
```
Comment (grace-y728) [inline client/src/App.js]: in the future there'll probably be a separate route to each different menu item, so we'll have to do that later on

File: client/src/components/StyledReviewComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+export const Rating = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin-left: 2%;
+`;
+
+export const StarRating = styled.div`
+    font-size: 30px;
+`;
```
Comment (grace-y728) [inline client/src/components/StyledReviewComponent.jsx]: for linters sake there should be a newline at the end of each file :)

File: package.json [hunk 3]
```diff
@@ -31,6 +31,7 @@
     "jsdoc": "^3.6.3",
     "mongoose": "^5.7.7",
     "react-router-dom": "^5.1.2",
+    "react-star-rating-component": "^1.4.1",
```
Comment (grace-y728) [inline package.json]: this should probably be put in the client/package.json since it's frontend?

Comment (vvndn) [reply package.json]: Just removed it from the package.json file, client/package.json already has it so we should be good

**Pull request merged**